### PR TITLE
change border color

### PR DESCRIPTION
**Pull request by vior1992**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
         body {
             color: white;
             background-color: orange;
-            border: 10px solid skyblue;
```
Comment (almuitur) [inline index.html]: Me gusta más mi color verde

File: index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
+            border: 10px solid #4bff03;
         }
     </style>
```
Comment (almuitur) [inline index.html]: Hey, move styles to file!

Comment (vior1992) [reply index.html]: Yep! That's true. But we must work at same place.

Comment (almuitur) [reply index.html]: No problem! I proceed to move styles to the file after merging.

**Pull request merged**